tests/probes/runlevel/test_probes_runlevel.sh

```diff
@@ -25,6 +25,9 @@ function test_probes_runlevel_A {
     require "egrep" || return 255
     require "awk" || return 255
 
+    local services_list=`get_services_list`
```

Comment:
Do you mind using the preferred `local services_list="$(get_services_list)"` syntax? iirc that's what `shellcheck` recommends.

Reply:
(However, the script does use backticks a lot so up to you).

Reply:
Yes, I'll change that.